testing/go/framework.go

```diff
@@ -160,10 +160,9 @@ func runScript(t *testing.T, ctx context.Context, script ScriptTest, conn *pgx.C
 
 				if assertion.Cols != nil {
 					fields := rows.FieldDescriptions()
-					if assert.Len(t, fields, len(assertion.Cols)) {
-						for i, col := range assertion.Cols {
-							assert.Equal(t, col, fields[i].Name)
-						}
+					assert.Len(t, fields, len(assertion.Cols))
```

Comment:
Doesn't this change allow for mismatched counts to cause a panic? If there are more `assertion.Cols` than `fields`, then `fields[i].Name` will panic right?

Reply:
Ah yeah it does panic if you have too many columns. The error message wasn't very clear when the lengths were off and trying to improve that. I changed this back and added a message to it instead

Reply:
I see, that makes sense!